README.md

```diff
@@ -484,6 +484,28 @@ require("onedarkpro").setup({
 })
 ```
 
+Alternatively, you can configure styles using individual highlight groups:
+
+```lua
+require("onedarkpro").setup({
+  highlights = {
+    ["@type"] = { italic = false, bold = false },
```

Comment:
Propose we keep these examples to a maximum of 3 otherwise we bloat the readme. We can direct the reader to the Neovim documentation regarding styles.

Also propose we use a combination of different highlight groups besides those starting with `@`.